src/pages/index.js

```diff
@@ -94,6 +210,9 @@ addCardButton.addEventListener("click", () => {
   cardAddFormPopup.open();
   formValidators["card-form"].toggleButtonState();
 });
+editAvatarButton.addEventListener("click", () => {
+  editAvatarPopup.open();
```

Comment:
* The avatar button should be also initially disabled  https://snipboard.io/7dM43X.jpg 


Please, call `toggleButtonState` here